src/internal/scheduler/AsyncScheduler.ts

```diff
@@ -1,7 +1,11 @@
 import { Scheduler } from '../Scheduler';
+import { Action } from './Action';
 import { AsyncAction } from './AsyncAction';
+import { SchedulerAction } from '../types';
+import { Subscription } from '../Subscription';
 
 export class AsyncScheduler extends Scheduler {
+  public static delegate?: Scheduler;
```

Comment:
While I'm open to alternatives, it turned out to be rather difficult to injection _just_ for AsyncScheduler because TestScheduler extends VirtualScheduler which also extends AsyncScheduler, so changes made here also apply to TestScheduler and many attempts I had ended in just infinite recursion 😆